### DISCUSSION TITLE
Running into an error when trying to Infer

### DISCUSSION THREAD

**Discussion by mobilerajs** [voicepaw/so-vits-svc-fork#630] [Q&A]
Hi,

While running SVCG, I'm receiving the following error:

"/Library/Frameworks/Python.framework/Versions/3.           
                    10/lib/python3.10/concurrent/futures/_base.py",             
                    line 403, in __get_result                                   
                        raise self._exception                                   
                    RuntimeError: don't know how to restore data                
                    location of torch.storage.UntypedStorage (tagged            
                    with mps:0)           

I believe this is related to the fact that I'm running on a Macbook M1. Any ideas? Thx.

**Comment by devcooch** [comment]
I am getting same error when running `crepe`, maybe try `dio`? Not sure if it must match model, I assume not
